pytest_tests/lib/helpers/node_management.py

```diff
@@ -68,14 +68,19 @@ def stop_storage_nodes(nodes: list[StorageNode]) -> None:
 
 
 @allure.step("Restart storage nodes")
-def restart_storage_nodes(nodes: list[StorageNode]) -> None:
+def restart_storage_nodes(neofs_env: NeoFSEnv) -> None:
     """
-    The function restarts specified storage nodes.
+    The function restarts all storage nodes of the specified neofs env.
     Args:
-       nodes: the list of nodes to restart
+       neofs_env: NeoFSEnv instance
     """
-    stop_storage_nodes(nodes)
-    start_storage_nodes(nodes)
+    for node in neofs_env.storage_nodes:
+        node.stop()
+    time.sleep(5)
```

Comment:
Doesn't seem to be reliable.

Reply:
replaced with an explicit wait for the process termination